aspnetcore/release-notes/aspnetcore-9.0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: What's new in ASP.NET Core 9.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 9.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 2/5/2024
+uid: aspnetcore-9
+---
+# What's new in ASP.NET Core 9.0
+
+This article highlights the most significant changes in ASP.NET Core 9.0 with links to relevant documentation.
+
+This article will be updated as new preview releases are made available.
+
+New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file.
```

Comment:
```suggestion
<!-- New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file. -->
```